unittests/test_schedulers.py

```diff
@@ -370,6 +370,26 @@ def test_no_empty_lines_in_preamble(minimal_job):
         assert line != ''
 
 
+def test_combined_access_constraint(make_job, slurm_only):
+    job = make_job(sched_access=['--constraint=c1'])
+    job.options = ['--constraint=c2,c3']
```

Comment:
Use the `-C` option here to exercise that path as well.

```suggestion
    job.options = ['-C c2,c3']
```